src/current/cockroachcloud/plan-your-cluster-serverless.md

```diff
@@ -0,0 +1,60 @@
+
+For optimal performance, your client application should be deployed in one of your cluster's configured regions. {{ site.data.products.serverless }} uses a geolocation routing policy to automatically route clients to the nearest region, even if that region is not one of your cluster's configured regions. This means that if you are running an application from an unconfigured region, you may be connected to that region and experience high network latency. This may be acceptable for development, but should be avoided for any production or performance-sensitive applications. Refer to the [{{ site.data.products.serverless }} FAQs](serverless-faqs.html#how-do-i-get-the-sql-endpoint-for-a-specific-region-of-my-multi-region-cluster) for information on overriding the automatic routing policy.
+
+Unlike {{ site.data.products.dedicated }} clusters, {{ site.data.products.serverless-plan }} clusters can have two regions if your goal is only to [survive zone failures](../{{site.versions["stable"]}}/multiregion-overview.html#survive-zone-failures). Clusters must have at least three regions to ensure that data replicated across regions can [survive region failures](../{{site.versions["stable"]}}/multiregion-overview.html#survive-region-failures).
+
+[Databases created](serverless-cluster-management.html#create-a-database) in {{ site.data.products.serverless }} will automatically inherit all of a cluster's regions, so it is not necessary to run [`ALTER DATABASE ... ADD REGION`](../{{site.versions["stable"]}}/alter-database.html#add-region) to configure regions when adding a database to the cluster. To override the default inheritance, you can specify the primary region with the [`CREATE DATABASE <db_name> WITH PRIMARY REGION`](../{{site.versions["stable"]}}/create-database.html) SQL syntax or the [`sql.defaults_primary_region`](../{{site.versions["stable"]}}/cluster-settings.html#setting-sql-defaults-primary-region) setting.
+
+For multi-region clusters, storage is billed at the same rate, but because data is replicated three times in the primary region and once in each additional region by default, storage costs will be multiplied based on the number of replicas. Write-heavy applications may experience a significant increase in RU consumption because more resources are required to replicate writes across all regions. Read-heavy applications may see a smaller increase in RU consumption because the resources required to read from a single region of a multi-region deployment are comparable to that of a single region deployment. During the [public preview](../{{site.versions["stable"]}}/cockroachdb-feature-availability.html#feature-availability-phases), RU usage for queries that cross regions will not account for inter-region bandwidth.
```

Comment:
> storage costs will be multiplied based on the number of replicas
This is misleading, b/c it sounds like if you have 5 replicas, you'd pay 5x the cost. But we include the cost of the first 3 replicas in the base cost, so you'd only pay 5/3 if you go from 3 => 5 replicas.

Reply:
tried to clarify this, lmk what you think